joint_teapot/utils/joj3.py

```diff
@@ -196,6 +196,11 @@ def generate_title_and_comment(
         + "[Joint-Teapot](https://github.com/BoYanZh/Joint-Teapot) with ❤️.\n"
     )
     for stage in stages:
+        stage_score = 0
+        for i, result in enumerate(stage["results"]):
```

Comment:
`enumerate` is not necessary here

also, consider `stage_score = sum(result["score"] for result in stage["results"])`

Reply:
better to check all scores are 0 to avoid a coincidence sum of 0.